core/src/dominio/Professor.java

```diff
@@ -231,24 +231,22 @@ public String getUsername() {
     }
 
     private void Avaliar(Aula aulaPedida, Interval primeiroDia, String motivo, List<Ausencia> ausencias) {
-        if(aulaPedida.getDiaDaSemana() == primeiroDia.getStart().getDayOfWeek())
+        if(aulaPedida.getDiaDaSemana() == primeiroDia.getStart().getDayOfWeek()
+                && aulaPedida.bateHorarioCom(primeiroDia)) 
         {
-            if(aulaPedida.bateHorarioCom(primeiroDia))
-            {
-                int anoAusência = primeiroDia.getStart().getYear();
-                int mêsAusência = primeiroDia.getStart().getMonthOfYear();
-                int diaAusência = primeiroDia.getStart().getDayOfMonth();
-                int inícioHoraAula = aulaPedida.getPeriodo().getStart().getHourOfDay();
-                int inícioMinutoAula = aulaPedida.getPeriodo().getStart().getMinuteOfHour();
-                int finalHoraAula = aulaPedida.getPeriodo().getEnd().getHourOfDay();
-                int finalMinutoAula = aulaPedida.getPeriodo().getEnd().getMinuteOfHour();
-                
-                DateTime inícioAusênciaAula = new DateTime(anoAusência,mêsAusência,diaAusência,inícioHoraAula,inícioMinutoAula);
-                DateTime finalAusênciaAula = new DateTime(anoAusência,mêsAusência,diaAusência,finalHoraAula,finalMinutoAula);
-                Interval períodoAusência = new Interval(inícioAusênciaAula,finalAusênciaAula);
-                Ausencia ausência = new Ausencia("0", períodoAusência, this, motivo, aulaPedida);
-                ausencias.add(ausência);
-            }
+            int anoAusência = primeiroDia.getStart().getYear();
+            int mêsAusência = primeiroDia.getStart().getMonthOfYear();
+            int diaAusência = primeiroDia.getStart().getDayOfMonth();
+            int inícioHoraAula = aulaPedida.getPeriodo().getStart().getHourOfDay();

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 87).](https://www.codacy.com/app/calebepb/substituicao/file/2736531173/issues/source?bid=3308597&fileBranchId=3351411#l240)